src/v_fmt.c

```diff
@@ -237,21 +239,55 @@ patch_t *V_LinearToTransPatch(const byte *data, int width, int height,
     return (patch_t *)output;
 }
 
-static patch_t *InvisibleSprite(int lump, pu_tag tag)
+static byte *FillCheckerBoardPattern(int width, int height)
 {
-    static const patch_t *patch;
-    static int size;
+    const int size = width * height;
 
-    if (!patch)
+    byte *buffer = malloc(size);
+    memset(buffer, v_darkest_color, size);
+
+    for (byte *p = buffer; p < buffer + size; p += 8)
     {
-        int num = (W_CheckNumForName)("TNT1A0", ns_sprites);
-        size = W_LumpLength(num);
-        patch = W_CacheLumpNum(num, PU_STATIC);
+        memset(p, v_lightest_color, 8);
+
+        if (((p - buffer) + 16) % (8 * 64) == 0)
+        {
+            p += 16;
+        }
+        else if (((p - buffer) + 8) % (8 * 64) != 0)
+        {
+            p += 8;
+        }
     }
 
+    return buffer;
+}
+
+static void *CheckerBoardFlat(int lump, pu_tag tag)
+{
+    static byte *buffer;
+
+    if (!buffer)
+    {
+        buffer = FillCheckerBoardPattern(64, 64);
+    }
+
+    const int size = 64 * 64;
     Z_Malloc(size, tag, &lumpcache[lump]);
-    memcpy(lumpcache[lump], patch, size);
+    memcpy(lumpcache[lump], buffer, size);
+    return lumpcache[lump];
+}
+
+static patch_t *CheckerBoardPatch(int lump, pu_tag tag)
+{
+    static byte *buffer;
 
+    if (!buffer)
+    {
+        buffer = FillCheckerBoardPattern(64, 128);
```

Comment:
@fabiangreffrath We don't have the texture size at v_fmt.c level, is 64x128 good enough? Maybe `-N0_TEX-` lump from PrBoom+ is simpler.

Reply:
He, but isn't the `-N0_TEX-` lump only used for missing flats?

Reply:
> He, but isn't the `-N0_TEX-` lump only used for missing flats?

Yes, the PrBoom+ solution is not good enough.

I don't understand how wall texture tiling works. Is there a way to force 64x64 texture tiling for all walls?

Reply:
Missing wall textures result in HOM. That's pretty much expected in Doom, so why reinvent the wheel?

Reply:
But what to do if the texture is in the wrong format (e.g. 32-bit PNG)? Bring back invisible sprite ("TNT1A0")?

Reply:
Or, do what we do now and error out with a clear message.

Reply:
Okay, I'll bring back `TNT1A0`. I think I was close, though:
![woof0001](https://github.com/fabiangreffrath/woof/assets/5077629/ca071dff-47b0-4d5b-8c52-8a1ca0af6bbe)

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1708/commits/e1e5f3871c8f487cb6676a0bc03236e3ad42132b